VirtoCommerce.QuoteModule.Test/QuoteModuleTests.cs

```diff
@@ -0,0 +1,408 @@
+using System;
```

Comment:
Not sure if we need to add this test at all. There is no much sense in adding such conversion tests. 
I would add test for TotalCalculators calculation logic. Maybe data prepared for TotalCalculator_ReturnsQuoteRequestTotals could be used for this.